node-operators/run-aut/index.md

```diff
@@ -22,26 +22,77 @@ To connect to a network and sync, get the genesis and bootnode files if needed,
     mkdir autonity-chaindata
     ```
 
-1. Start autonity:
+2. Generate the `autonitykeys` private key file for your node. This must be performed on the host machine running the Autonity Go Client, using the `autonity genAutonityKeys --writeaddress` command:
+
+	```bash
+	./autonity genAutonityKeys --writeaddress ./<DIR_PATH>/<OUT_KEY_FILE_NAME>
+	```
+
+	where:
+
+	- `<DIR_PATH>`: is the path to the directory where you will store the private key file of the node.
+	- `<OUT_KEY_FILE_NAME>`: is the name of the P2P autonity keys private key file you are generating for your node.
+
+	For example, running the command where the directory is `(pwd)/keystore` and the file name `myAutonityKeysFile`:
+
+	```bash
+	./autonity/build/bin/autonity genAutonityKeys --writeaddress ./keystore/myAutonityKeysFile
+	```
+
+	You should see something like this output to your terminal:
+
+	```bash
+	Node address: 0x0f083e047725bF1dFf3efe110430B98AAac55986
+	Node public key: 0xee0f1195d19b422cd6fc69b060853b7c21094a84e66a225b549afafe7ba44996f85015d6a7f19d32e1e978db26db4083c8db8e153ddea6f1aa4c65608374c868
+	Consensus public key: 0x8aa83a28e235072ffdae48ff01ccc46e2b8d9dc16df9b6ff87ffa5ff6d8f90a2852649a60563237cd66a256f60a92e69
+	```
+
+::: {.callout-important title="Important" collapse="false"}
+Remember to backup your `autonitykeys` file! Copy it to a safe location!
+:::
+
+::: {.callout-important title="Important" collapse="false"}
+Copy the output - you will need the output if you intend to register your validator as a node.
+
+It is *not* possible to inspect the keyfile to retrieve the `Consensus public key` which you MUST provide as part of the validator registration information.
```

Comment:
We will have an autonityKeys inspector in tool `ethkey` to retrieve it, but it is not in this release.

Reply:
Thanks. I have edited to correct this and add in the `autinspect` command from the various fixes PR - https://github.com/autonity/autonity/pull/943

See commit e5e1b19d1dfd65cb8201f6f4256ebd5099c9539d